.github/workflows/tests.yml

```diff
@@ -125,13 +125,13 @@ jobs:
           CDR_DB_HOST: db
           CDR_DB_PORT: 5432
 
-#https://github.com/coverallsapp/github-action/issues/29
-#      - name: Coveralls Parallel
-#        uses: coverallsapp/github-action@master
-#        with:
-#          github-token: ${{secrets.github_token}}
-#          flag-name: run-${{matrix.ci_node_index}}
-#          parallel: true
+      - name: SimpleCov Parallel
+        uses: coverallsapp/github-action@master
+        with:
+          path-to-lcov: "./coverage/lcov.info"
+          github-token: ${{secrets.github_token}}
+          flag-name: run-${{matrix.ci_node_index}}
+          parallel: true
 
```

Comment:
**finalize-coveralls** also should be uncommented, because we are using parrallelization

Reply:
@dmitry-sinina ok, I will do this tomorrow.